build-scripts/build-dnsdist-rpm

```diff
@@ -193,8 +196,8 @@ fi
 
 %postun
 %if 0%{?el6}
-if [ "\$1" -ge "1" ] ; then
-  /sbin/service %{name} condrestart >/dev/null 2>&1 || :
+if [ -x /sbin/initctl ] && /sbin/initctl status %{name} 2>/dev/null | grep -q 'running' ; then
+  /sbin/initctl stop %{name} > /dev/null 2>&1 || :
```

Comment:
given this swallows all errors and it's a stop action, i don't see much point in calling initctl status first...